<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>tab1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="testText">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="plainText">
       <string>test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backbutton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>backupdate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>tab2</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>331</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>tab3</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>tab4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab5">
     <attribute name="title">
      <string>tab5</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab6">
     <attribute name="title">
      <string>页</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>501</width>
        <height>191</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>33</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>44</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>cc</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>aa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dd</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>tt</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="urlLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QComboBox" name="choose1box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>国际</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>体育</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>科技</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>测试</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="okbutton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>271</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="newAction"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu01">
    <property name="title">
     <string>01</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu02">
    <property name="title">
     <string>02</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu03">
    <property name="title">
     <string>03</string>
    </property>
    <addaction name="newAction"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu01"/>
   <addaction name="menu02"/>
   <addaction name="menu03"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="newAction">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
